packages/rancher-logging/overlay/templates/loggings/aks/logging.yaml

```diff
@@ -15,6 +15,9 @@ spec:
     inputTail:
       Tag: "aks"
       Path: "/var/log/azure/kubelet-status.log"
+    security:
+      podSecurityPolicyCreate: true
+      roleBasedAccessControlCreate: true
```

Comment:
Is there value in exposing this with values.yaml?

```yaml
{{- with .Values.security }}
security: # insert values here
{{- end }}
```

Reply:
I can't think of any reason why you would want to deactivate it. If a cluster does not use PodSecurityPolicies, it does not matter if the objects are there or not, they are just ignored.
Theoretically someone could want to use their own PodSecurityPolicy instead of the one provided by the operator, but I can't think of any value that would give you.
But maybe I'm missing something here.

Reply:
@bashofmann I'm inclined to agree. We can always add it later then.

Reply:
Reading upstream, these 2 just makes a pre-baked fluentD PSP ?
I think only reason for shutting it off would be if it breaks something, but also I could see down the line PSP's coming with a warning deprecation message.